Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -125,17 +125,18 @@ protected void LoadApplication(Application application)
 				_renderersAdded = true;
 			}
 
-			if (application == null)
-				throw new ArgumentNullException("application");
-
-			_application = application;
-			(application as IApplicationController)?.SetAppIndexingProvider(new AndroidAppIndexProvider(this));
+			_application = application ?? throw new ArgumentNullException(nameof(application));
+			(application as IApplicationController).SetAppIndexingProvider(new AndroidAppIndexProvider(this));
 			Xamarin.Forms.Application.SetCurrentApplication(application);
 
-			SetSoftInputMode();
+			if (Xamarin.Forms.Application.Current.OnThisPlatform().GetShouldSetWindowSoftInputModeAtStartup())
+				SetSoftInputMode();
+			else
+				Xamarin.Forms.Application.Current.OnThisPlatform().UseWindowSoftInputModeAdjust(WindowSoftInputModeAdjust.Unspecified);
```

Comment:
Default to `Unspecified` since XF is currently defaulting to `Pan` and we do not want that.